lib/browser/api/dialog.js

```diff
@@ -175,6 +181,12 @@ module.exports = {
       throw new TypeError('Message must be a string')
     }
 
+    if (securityScopedBookmarks == null) {
```

Comment:
It can be true only in two cases. If `securityScopedBookmarks` is `undefined`, or it's `null`.

`null` is obviously an invalid value, you should throw an error in this case.
`undefined` means it is not provided, you can simply use default value in the destructuring assignment:

```js
let {securityScopedBookmarks = false} = options
```